test/SniffTestCase.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-coding-standard for the canonical source repository
+ * @copyright https://github.com/zendframework/zend-coding-standard/blob/master/COPYING.md Copyright
+ * @license   https://github.com/zendframework/zend-coding-standard/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\CodingStandard;
+
+use PHPUnit\Framework\Constraint\IsEqual;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Util\InvalidArgumentHelper;
+
+class SniffTestCase extends TestCase
+{
+    public function processAsset($asset)
+    {
+        $phpcs = new \PHP_CodeSniffer();
+
+        $standard = 'src/ZendCodingStandard/ruleset.xml';
+        $options = array_merge($phpcs->cli->getDefaults(), [
+            'encoding'    => 'utf-8',
+            'files'       => [$asset],
+            'standard'    => $standard,
+            'showSources' => true,
+        ]);
+
+        $reflection = new \ReflectionProperty($phpcs->cli, 'values');
+        $reflection->setAccessible(true);
+        $reflection->setValue($phpcs->cli, $options);
+
+        $phpcs->initStandard($standard, ['ZendCodingStandard.Commenting.FileLevelDocBlock']);
+        $phpcs->setIgnorePatterns([]);
+
+        return $phpcs->processFile($asset);
+    }
+
+    public function assertErrorCount($expectedCount, \PHP_CodeSniffer_File $file)
+    {
+        if (! \is_int($expectedCount)) {
```

Comment:
Why is this globally qualified?

I actually have no issues with globally qualifying functions and constants. However, choose one or the other, and stick with it throughout. This one jumped at me because I haven't seen it done anywhere else.